meson.build

```diff
@@ -45,8 +45,12 @@ else
 endif
 
 cy = meson.get_compiler('cython')
+cdata = configuration_data()
```

Comment:
Can you move the configuration_data next to the `configure_file` call? I assume here you did it to avoid repeating the `cy.version()` check, but I think it reads easier to keep this all together